### PR TITLE
* Core/Conversation Allow to define 'nearest creature id' for convers…

### PR DESCRIPTION
**Pull request by Mistixx**
…ation actors

**Changes proposed:**

-  
-  
-  

**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [ ] master

**Issues addressed:**

Closes #  (insert issue tracker number)

**Tests performed:**

(Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by MaxtorCoder** [comment]
What real usage has this? Example?

**Comment by illfated** [comment]
Looks like the PR owner has not taken the time to fill in the PR comment template. Could it be that this PR targets incorrect repository?

File: src/server/game/Globals/ConversationDataStore.cpp [hunk 1]
```diff
@@ -106,7 +107,8 @@ void ConversationDataStore::LoadConversationTemplates()
             uint32 conversationId         = fields[0].GetUInt32();
             uint32 actorId                = fields[1].GetUInt32();
             ObjectGuid::LowType actorGuid = fields[2].GetUInt64();
-            uint16 idx                    = fields[3].GetUInt16();
+            uint32 actorNearId            = fields[3].GetUInt32();
+            uint16 idx = fields[4].GetUInt16();
```
Comment (matanshukry) [inline src/server/game/Globals/ConversationDataStore.cpp:111]: indent

Comment (illfated) [reply src/server/game/Globals/ConversationDataStore.cpp:111]: Yes, preferably keep the codestyle from the lines above.

**Comment by matanshukry** [comment]
@Mistixx how is the SC script loader relevant?
Also, more importantly - can you fill in the relevent details and describe what is the use case for this?

I also have an issue in this area, and it would be better to think of a solution that applies to both use cases.

**Comment by MaxtorCoder** [comment]
> 
> 
> @Mistixx how is the SL script loader relevant?
> Also, more importantly - can you fill in the relevent details and describe what is the use case for this?
> 
> I also have an issue in this area, and it would be better to think of a solution that applies to both use cases.

It totally isn't relevant at all.

**Comment by Mistixx** [comment]
I'm sorry is that I accidentally made a commit to another repository and had the path set for trinity

**Comment by Mistixx** [comment]
It has been an error so you can delete this pull request

**Comment by MaxtorCoder** [comment]
> 
> 
> It has been an error so you can delete this pull request

You can close it yourself as well.

**Pull request closed without merging**